### PR TITLE
[TC-9117] Add tXML order webhook

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-9117

Please review https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/Z6andW8snaNud1yWXi3k

### Scope
This PR adds the `tXML` order webhook

### Submitter pre-flight check

- [ ] Review your documentation
- [ ] Add labels `needs reviewing`
- [ ] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: api/webhook-vs-polling.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ When using `POST` the order webhook request body contains:
 * `orderDocumentsEvent`: The order documents event, when using documents.
 
 {% hint style="info" %}
-The [POST Order Webhook](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost) endpoint [XML](json-vs-xml.md#xml) support is under development.
+The [POST Order Webhook](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost) endpoint supports [XML](json-vs-xml.md#xml).
```
Comment (roy-tc) [inline api/webhook-vs-polling.md:36]: "Both JSON and XML" ?

File: order/buyer/receive/README.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ If you choose the POST webhook API, you may choose between the native or simple
 
 {% page-ref page="../../../api/delivery-schedule.md" %}
 
+{% hint style="info" %}
+The [POST order webhook](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost) endpoint supports [XML](../../../api/json-vs-xml.md#xml).
```
Comment (roy-tc) [inline order/buyer/receive/README.md:22]: "Both JSON and XML"?

File: order/supplier/receive/README.md [hunk 3]
```diff
@@ -18,6 +18,10 @@ If you choose the POST webhook API, you may choose between the native or simple
 
 {% page-ref page="../../../api/delivery-schedule.md" %}
 
+{% hint style="info" %}
+The [POST order webhook](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost) endpoint supports [XML](../../../api/json-vs-xml.md#xml).
```
Comment (roy-tc) [inline order/supplier/receive/README.md:22]: Both JSON and XML

**Review by roy-tc — COMMENTED** [review]
Looks good, left a couple of suggestions

File: api/webhook-vs-polling.md (hunk 1 above)
Comment (marcmatt) [reply api/webhook-vs-polling.md:36]: Updated

File: order/buyer/receive/README.md (hunk 2 above)
Comment (marcmatt) [reply order/buyer/receive/README.md]: Removed this link

File: order/supplier/receive/README.md (hunk 3 above)
Comment (marcmatt) [reply order/supplier/receive/README.md]: Removed this link

**Review by roy-tc — APPROVED** [review]

**Pull request merged**